src/foam/u2/ExportModal.js

```diff
@@ -115,12 +129,19 @@ foam.CLASS({
         return;
       }
 
+      var filteredColumnsCopy = this.filteredTableColumns;
+      if ( this.exportAllColumns )
+        this.filteredTableColumns$.set(null);
```

Comment:
Couldn't you just do this.filteredTableColumns = null; ?

Reply:
My mistake